osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineBreakDisplay.cs

```diff
@@ -71,7 +71,10 @@ private void recreateBreaks()
                 if (!shouldBeVisible(breakPeriod))
                     continue;
 
-                Add(new TimelineBreak(breakPeriod));
+                Add(new TimelineBreak(breakPeriod)
+                {
+                    OnDeleted = b => breaks.Remove(b),
```

Comment:
I believe this should be accompanied by a `changeHandler.SaveState` call for undo/redo/"do you want to save your changes" functionalities.